tests/acceptance/features/webUIFilesCopy/copy.feature

```diff
@@ -69,38 +85,48 @@ Feature: copy files and folders
 
   @issue-ocis-reva-243
   Scenario: copy a file into another folder with no change permission
-    Given user "Brian" has been created with default attributes
+    Given user "Alice" has created file "lorem.txt"
+    And user "Alice" has created folder "simple-folder"
+    And user "Brian" has been created with default attributes and without skeleton files
+    And user "Brian" has created folder "simple-folder"
     And user "Brian" has shared folder "simple-folder" with user "Alice" with "read" permissions
     And user "Alice" has logged in using the webUI
-    When the user tries to copy file "lorem.txt" into folder "simple-folder (2)" using the webUI
-    Then as "Alice" file "simple-folder (2)/lorem (2).txt" should not exist
+    When the user tries to copy file "lorem.txt" into folder "simple-folder" using the webUI
+    Then as "Alice" file "simple-folder/lorem.txt" should not exist
 
   @issue-ocis-reva-243
   Scenario: copy a folder into another folder with no change permission
-    Given user "Brian" has been created with default attributes
+    Given user "Alice" has created folder "simple-empty-folder"
+    And user "Brian" has been created with default attributes and without skeleton files
+    And user "Brian" has created folder "simple-folder"
     And user "Brian" has shared folder "simple-folder" with user "Alice" with "read" permissions
     And user "Alice" has logged in using the webUI
-    When the user tries to copy folder "simple-empty-folder" into folder "simple-folder (2)" using the webUI
-    Then as "Alice" file "simple-folder (2)/simple-empty-folder (2)" should not exist
+    When the user tries to copy folder "simple-empty-folder" into folder "simple-folder" using the webUI
+    Then as "Alice" file "simple-folder/simple-empty-folder" should not exist
 
 
   Scenario: copy a folder into the same folder
-    Given user "Alice" has logged in using the webUI
+    Given user "Alice" has created folder "simple-empty-folder"
+    And user "Alice" has logged in using the webUI
     When the user tries to copy folder "simple-empty-folder" into folder "simple-empty-folder" using the webUI
     Then the error message with header 'An error occurred while copying simple-empty-folder' should be displayed on the webUI
     And as "Alice" file "simple-empty-folder/simple-empty-folder" should not exist
 
 
   Scenario: copy a folder into another folder with same name
-    Given user "Alice" has logged in using the webUI
-    When the user copies folder "simple-empty-folder" into folder "folder with space/simple-empty-folder" using the webUI
-    Then breadcrumb for folder "simple-empty-folder" should be displayed on the webUI
+    Given user "Alice" has created folder "simple-empty-folder"
+    And user "Alice" has created folder "folder with space"
+    And user "Alice" has logged in using the webUI
+    When the user copies folder "simple-empty-folder" into folder "folder with space" using the webUI
```

Comment:
imo as the scenario description folder `folder with space` has to have folder `simple-empty-folder` otherwise there wouldn't be folder with same name